webview/src/shared/components/messagesMenu/styles.module.scss

```diff
@@ -25,6 +25,14 @@
   text-overflow: ellipsis;
 }
 
+.disabledItem {
+  opacity: $disabled-opacity;
+
```

Comment:
Should we be adding a `pointer: auto` or `pointer: default` to disabled items?

Reply:
Changed both to `cursor: default;` to match the VS Code context menu.